crates/lsp/src/lib.rs

```diff
@@ -198,6 +205,14 @@ impl<L: LSPLang> LanguageServer for Backend<L> {
       .await;
     Ok(self.on_code_action(params).await)
   }
+
+  async fn execute_command(&self, params: ExecuteCommandParams) -> Result<Option<Value>> {
+    self
+      .client
+      .log_message(MessageType::INFO, "execute command!")
```

Comment:
can we have a more descriptive message?

Reply:
it is copied from the code of code_action
```rust
  async fn code_action(&self, params: CodeActionParams) -> Result<Option<CodeActionResponse>> {
    self
      .client
      .log_message(MessageType::INFO, "run code action!")
      .await;
    Ok(self.on_code_action(params).await)
  }
```
more descriptive message is shown here

https://github.com/SoonIter/ast-grep/blob/9206a9c154e7d7bd34345424753257bb649a3517/crates/lsp/src/lib.rs#L463-L469